addons/l10n_es/data/template/account.fiscal.position-es_common.csv

```diff
@@ -61,7 +61,7 @@
 "","","","","","","","","","","account_common_7030","account_common_7032"
 "","","","","","","","","","","account_common_7040","account_common_7042"
 "","","","","","","","","","","account_common_7050","account_common_7052"
-"fp_not_subject_tai","","Regime not subject to localition rules (TAI, Canarias, ...)","Régimen No sujeto por reglas de localización (TAI - Canarias, Ceuta, Melilla...)","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_iva0_ns_b","",""
+"fp_not_subject_tai","","Régimen No sujeto por reglas de localización (TAI)","Régimen No sujeto por reglas de localización (TAI - Canarias, Ceuta, Melilla...)","","","","","account_tax_template_p_iva4_bc","account_tax_template_p_iva0_ns_b","",""
```

Comment:
would it not make sense to keep (but fix) the English, since we have the translated term